examples/db-templates/mariadb-ephemeral-template.json

```diff
@@ -156,16 +156,16 @@
     },
     {
       "name": "MYSQL_USER",
-      "displayName": "MySQL User",
+      "displayName": "MySQL Connection Username",
       "description": "Username for MariaDB user that will be used for accessing the database.",
       "generate": "expression",
       "from": "user[A-Z0-9]{3}",
       "required": true
     },
     {
       "name": "MYSQL_PASSWORD",
-      "displayName": "MySQL Password",
-      "description": "Password for the MariaDB user.",
+      "displayName": "MySQL Connection Password",
+      "description": "Password for the MariaDB connection user.",
```

Comment:
noticed this says MariaDB, here and in the description above, old copy/paste error?